packages/worker/src/migrations/1730806000905-TxReceiptRootColumnNullable.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class TxReceiptRootColumnNullable1730806000905 implements MigrationInterface {
+  name = "TxReceiptRootColumnNullable1730806000905";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "transactionReceipts" ALTER COLUMN "root" DROP NOT NULL`);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "transactionReceipts" ALTER COLUMN "root" SET NOT NULL`);
```

Comment:
Will this work if there will already be null fields in the database? Shouldn't we have something like `DEFAULT '\\0x00..00'::bytea` or smth like this?

Reply:
Yes, it will, I checked it.